pipelines/pipeline_katello_nightly.yml

```diff
@@ -11,35 +11,42 @@
   roles:
     - forklift
 
-- hosts: pipeline-katello-nightly-centos7
+- hosts:
+  - pipeline-katello-nightly-centos7
+  - pipeline-proxy-nightly-centos7
   become: yes
   vars:
     katello_repositories_version: nightly
     foreman_repositories_version: nightly
     katello_repositories_use_koji: true
+    foreman_installer_skip_installer: true
+  roles:
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - update_os_packages
+    - foreman_installer
```

Comment:
The roles are executed serially but in parallel across 2 hosts.  update_os_packages on both happen concurrently.  Then foreman_installer.  This is a pretty significant savings in time to run the pipeline.

What's the increase in time to fail you're seeing here?